Solution of the task "Work With Ssv File"

DanaKhromenko opened the pull request:
- Added class OperationTable. It will holds tha data from WorkWithFile class in table format;
- Implemented class WorkWithFile where I read data from file, convert data to table, sum data up and write to file.

DanaKhromenko commented:
Fixed Travis's errors:
- line is longer than 100 characters;
- using the '.*' form of import should be avoided - java.io.*.

RomanDubovskyi commented on src/main/java/core/basesyntax/WorkWithFile.java:
```diff
@@ -1,8 +1,70 @@
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String OPERATION_SEPARATOR = ",";
```
ця зміння дублюється в двох класах, зроби її публічною і реюзни в іншому класі

RomanDubovskyi commented on src/main/java/core/basesyntax/OperationTable.java:
```diff
@@ -0,0 +1,76 @@
+package core.basesyntax;
+
+public class OperationTable {
```
nice oop aproach, але враховуючи, що ми всеодно впираємось в конкретні операції buy / supply і додаткової гнучкості коду це не дає,  я б це спростив після зчитування файла до такого до такого: 
```suggestion
                String[] lineElements = line.split(CSV_SEPARATOR);
                if (lineElements[OPERATION_INDEX].equals(BUY_OPERATION)) {
                    buyAmount += Integer.parseInt(lineElements[AMOUNT_INDEX]);
                } else {
                    supplyAmount += Integer.parseInt(lineElements[AMOUNT_INDEX]);
                }
```
P.S. 
it's up to you, можеш залишити поточний підхід

RomanDubovskyi commented on src/main/java/core/basesyntax/WorkWithFile.java:
```diff
@@ -1,8 +1,70 @@
 
+    private void writeToFile(String toFileName, String resultText) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName, true))) {
+            for (int i = 0; i < resultText.length(); i++) {
+                bufferedWriter.append(resultText.charAt(i));
+            }
```
```suggestion
            bufferedWriter.write(resultText);
```

DanaKhromenko commented:
- replaced "LINE_SEPARATOR" with "System.LineSeparator()";
- replaced constant OPERATION_SEPARATOR in OperationTable class with same constant from WorkWithFile;
- replaced BufferedWriter's loop appending with "bufferedWriter.write(...)".
